### DISCUSSION TITLE
Permanent Drawer

### DISCUSSION THREAD

**Discussion by tianhuat** [quasarframework/quasar#11315] [General - Components / Directives / etc]
Is there a way to make QDrawer shows permanently, and hide the  hamburger icon (in red rectangle)? Or there are other components that could be used for this purpose?

[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/UgGJc.png

**Comment by MilosPaunovic** [comment]
Yes, you can do that.
[https://codepen.io/milospaunovic/pen/gOxBYGX?editors=101](https://codepen.io/milospaunovic/pen/gOxBYGX?editors=101)
